ansible/tasks/postgres-extensions/24-pgroonga.yml

```diff
@@ -1,4 +1,11 @@
 # groonga and pgroonga
+- name: groonga - download & install dependencies
+  apt:
+    pkg:
+      - libmecab2
```

Comment:
I think that `libmecab-dev` is needed to build PGroonga's MeCab support.
`libmecab2` doesn't provide header files.

Reply:
Good catch - thanks!

I picked those libraries using Ubuntu 20's [`groonga-tokenizer-mecab`'s dependency list](https://packages.ubuntu.com/focal/groonga-tokenizer-mecab) so that PGroonga's buildchain detects those and MeCab support is turned on during compilation. 

This did enable MeCab support in the resulting build. I agree, though, that including `libmecab-dev` to increase the stability and consistency of future builds is desirable.

Just added the lib, also works considering that `libmecab2` is [its only dependency](https://packages.ubuntu.com/focal/libmecab-dev).

Reply:
Ah, we already installed `libmecab-dev` in `ansible/tasks/docker/setup.yml`.
So we don't need `libmecab-dev` here. (We don't need `libmecab2` too because it's automatically installed via `libmecab-dev`.)

We may want to move remained `mecab-naist-jdic` to `ansible/tasks/docker/setup.yml`.

Reply:
Ah, I see! Was not aware of the MeCab-related packages also being defined in `ansible/tasks/docker/setup.yml` - thanks again!

I shall move them to `ansible/tasks/postgres-extensions/24-pgroonga.yml` as ideally we'd want to have all the dependencies of an extension contained within the extension's Ansible task, as we're building both Docker images for local development and self-hosting, as well as Amazon AMIs for the Supabase platform to use.
Also removing `libmecab2` from the task file.